sdk/examples/noop_go/src/sawtooth_noop/handler/handler.go

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2017 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ------------------------------------------------------------------------------
+ */
+
+package handler
+
+import (
+	"crypto/sha512"
+	"encoding/hex"
+	"sawtooth_sdk/logging"
+	"sawtooth_sdk/processor"
+	"sawtooth_sdk/protobuf/processor_pb2"
+	"strings"
+)
+
+var logger *logging.Logger = logging.Get()
+
+type NoopHandler struct {
+	namespace string
+}
+
+func NewNoopHandler(namespace string) *NoopHandler {
+	return &NoopHandler{
+		namespace: namespace,
+	}
+}
+
+var FAMILY_NAME = "noop"
```

Comment:
If you make it const, I don't see a reason to keep it private. Here is an example of using all caps for consts https://golang.org/doc/effective_go.html#constants